app/services/katello/candlepin/consumer.rb

```diff
@@ -0,0 +1,29 @@
+module Katello
+  module Candlepin
+    class Consumer
+      include LazyAccessor
+
+      attr_accessor :uuid
+
+      def initialize(uuid)
+        self.uuid = uuid
+      end
+
+      def regenerate_identity_certificates
+        Rails.logger.debug "Regenerating consumer identity certificates: #{name}"
+        Resources::Candlepin::Consumer.regenerate_identity_certificates(self.uuid)
+      rescue => e
+        Rails.logger.debug e.backtrace.join("\n\t")
```

Comment:
Should we error log this? Thoughts on whether we should have a custom logger for this layer?